seatunnel-connectors-v2/connector-cdc/connector-cdc-base/src/main/java/org/apache/seatunnel/connectors/cdc/base/source/enumerator/HybridSplitAssigner.java

```diff
@@ -114,7 +114,8 @@ public Optional<SourceSplitBase> getNext() {
 
     @Override
     public boolean waitingForCompletedSplits() {
-        return snapshotSplitAssigner.waitingForCompletedSplits();
+        return snapshotSplitAssigner.waitingForCompletedSplits()
+                || incrementalSplitAssigner.waitingForAssignedSplits();
```

Comment:
Should we keep the call named incrementalSplitAssigner.waitingForCompletedSplits() ?

Reply:
will use it in the future